e2e/utils_test.go

```diff
@@ -0,0 +1,41 @@
+package e2e_test
+
+import (
+	"fmt"
+
+	"k8s.io/client-go/kubernetes"
+)
+
+func runOnAllClusters(testContext *TestContext, f func(kubernetes.Interface) error) error {
+	var err error
+
+	err = f(testContext.HubClient())
+	if err != nil {
+		fmt.Printf("Failed to run function on hub cluster: %v\n", err)
+	}
```

Comment:
We want to support environment without a hub - drenv and ramenctl supports this now.